client/src/views/page-components/account/history/history.ts

```diff
@@ -0,0 +1,99 @@
+import { $ } from '@src/utils/helper';
+import { createDOMWithSelector, matchCategoryAndImg } from '@src/utils/helper';
+import './history.scss';
+
+export default class History {
+  state: any;
+  history: HTMLElement;
+  constructor({ parent, state }) {
+    this.history = createDOMWithSelector('div', '.history');
+
+    parent.appendChild(this.history);
+    this.setProperty(state);
+    this.render();
+  }
+
+  setProperty(state): void {
+    this.state = state;
+  }
+
+  render(): void {
+    $('.history').innerHTML = `
+        ${this.createHistoryHeader()}
+        ${this.createHistoryContent()}
+        `;
+  }
+
+  createHistoryHeader(): string {
+    return `
+      <div class='history__header'>
+        <div class='history__header-left'>
+          <span class='history__text'>내역</span>
+          <span class='history__add'>+</span>
+        </div>
+        <div class='history__header-right'>
+          <span>수입</span>  
+          <span>${this.state.income}</span>
+          <span>지출</span>
+          <span>${this.state.expenditure}</span>
```

Comment:
이렇게 사용하신다면 굳이 일일히 구분 안해도 될것 같습니다!
여담이지만, 수입 지출은 각각 색깔을 다르게 할 생각이라 구분하는게 좋겠네요 :)

```suggestion
          <span>수입 ${this.state.income}</span>  
          <span>지출 ${this.state.expenditure}</span>
```

Reply:
오호 좋습니다 반영할게요~!